cumulus_library/study_parser.py

```diff
@@ -1,51 +1,14 @@
-"""Contains classes for loading study data based on manifest.toml files"""
+"""Class for loading study configuration data from manifest.toml files"""
 
-import contextlib
-import csv
-import importlib.util
-import inspect
 import pathlib
-import sys
-import typing
 
-import pandas
-import pytablewriter
 import toml
-from rich.progress import Progress, TaskID, track
 
-from cumulus_library import (
-    __version__,
-    base_table_builder,
-    base_utils,
-    databases,
-    enums,
-    errors,
-    protected_table_builder,
-)
-from cumulus_library.statistics import psm
-from cumulus_library.template_sql import base_templates
-
-
-@contextlib.contextmanager
-def _temporary_sys_path(add: pathlib.Path) -> None:
-    orig_path = list(sys.path)
-    try:
-        sys.path.insert(0, str(add))
-        yield
-    finally:
-        sys.path = orig_path
+from cumulus_library import errors
 
 
 class StudyManifestParser:
```

Comment:
tiny nit: it's a little verbose to me to see this `Parser` terminology all around the place. Like, yes this parses a manifest as an implementation detail, but the object itself represents the result - you're really passing around a `StudyManifest` object from function to function. When I see an object or method with a parser name, I kind of expect it to be stage 1 that then spits out a Manifest object result (or just have the parsing be a quiet detail of the class - like `.from_file(path)` or `__init__(path)`

Reply:
hmm - not the way my brain usually works, but i see your point. That's a breaking change, though, so i'll put an issue in the 3.0 bucket for it.